supportedBackends/jes/src/test/scala/cromwell/backend/impl/jes/statuspolling/JesApiQueryManagerSpec.scala

```diff
@@ -97,10 +97,6 @@ class JesApiQueryManagerSpec extends TestKitSuite("JesApiQueryManagerSpec") with
         val probe = statusRequesters(index)
         probe.expectNoMsg(max = AwaitAlmostNothing)
       }
-      (batchOffset until batchOffset + BatchSize) foreach { index =>
-        val probe = statusRequesters(index)
-        probe.expectMsg(max = TestExecutionTimeout, hint = s"Polling error to requester #$index", obj = JesPollingActor.JesPollError)
-      }
```

Comment:
We should also check the queue is appropriately refreshed. Maybe as part of the eventually on line 110? If I'm reading the test correctly:
```
eventually { 
  ...
  jaqmActor.underlyingActor.workQueue.size should be(2 * BatchSize - batchOffset) }
```
Also while you're here I'm pretty sure we need to fix some lines:
- `[90] ... sender = statusPoller2`
- `[93] stopMethod(statusPoller2)`